config/initializers/new_framework_defaults_7_2.rb

```diff
@@ -0,0 +1,71 @@
+# typed: false
```

Comment:
BTW I'm intrigued by this. Are you using sorbet somehow? I see `standard-sorbet` in the Gemfile, but no actual sorbet.

Reply:
It has been on my maybe-one-day fun project list, but no, not yet.